docs/contributing-analyzer.md

```diff
@@ -0,0 +1,59 @@
+# Building, Testing and Debugging the Analyzer
+
+All C# and VB.NET analyzers present in SonarLint for Visual Studio, in the SonarQube SonarC# Plugin or in the
+SonarQube SonarVB plugin are being developed here.
+These analyzers rely on Roslyn 1.0 API.
+
+## Working with the code
```

Comment:
I recommend to consistently add a blank line after section headings. It makes it easier to read as plain text (which is one perfectly reasonable way to consume markdown)